.semver

```diff
@@ -1 +1 @@
-2.3.0
+2.4.0
```

Comment:
minor: major change for the library?

Reply:
Yeah, so semver for me is breaking:feature:fix

We've added a feature with breaking existing behaviour (or believe we have :)) so bump the feature version